tests/functional/adapter/tidb/utils/test_util.py

```diff
@@ -154,3 +148,11 @@ class TestSplitPart(BaseSplitPart):
 
 class TestStringLiteral(BaseStringLiteral):
     pass
+
+
+class TestCurrentTimestampTiDB(BaseCurrentTimestamp):
+    pass
+
+
+class TestCurrentTimestampNaiveTiDB(BaseCurrentTimestampNaive):
+    pass
```

Comment:
Can we add these test: BaseArrayAppend, BaseArrayConcat, BaseArrayConstruct.
Can we add BaseInvalidGrants in grant file folder

Reply:
TiDB does not support `Array` type.
For `BaseInvalidGrants`, I will add it.